src/app/confirm/success/page.tsx

```diff
@@ -0,0 +1,22 @@
+"use client";
+
+import { useSearchParams } from "next/navigation";
+import React from "react";
+
+const SuccessPage = () => {
+  const params = useSearchParams();
+
+  return (
+    <div className="text-center">
+      <p className="my-8 text-8xl">👍</p>
+      <h1 className="text-xl">
+        Thanks for letting us know about your decision!{" "}
```

Comment:
same comment as above...could just say "your spot has been confirmed". or "you have been removed from the carpool"